packages/db/package.json

```diff
@@ -13,14 +13,18 @@
   "types": "./index.ts",
   "license": "MIT",
   "scripts": {
-    "build": "esbuild migrate.ts --bundle --platform=node --outfile=migrate.cjs",
+    "build": "pnpm run build:sqlite && pnpm run build:mysql",
+    "build:sqlite": "esbuild migrations/sqlite/migrate.ts --bundle --platform=node --outfile=migrations/sqlite/migrate.cjs",
+    "build:mysql": "esbuild migrations/mysql/migrate.ts --bundle --platform=node --outfile=migrations/mysql/migrate.cjs",
```

Comment:
Shouldn't the CI be changed too if this creates 2 distinct versions of the code ?

Reply:
What do you mean with the ci?

Reply:
The GitHub Action used to deploy docker images / build the app for the different releases

Reply:
Not sure but the way we currently have done it both databases are supported from the same docker container

Reply:
Perfect then